Introduce new Candlepin changes (#1932)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-organization.adoc

. In the *Label* field, enter a unique identifier for the organization.
This is used for creating and mapping certain assets, such as directories for content storage.
Use letters, numbers, underscores, and dashes, but no spaces.

> ifdef::satellite[]

Reviewer: I am following internal discussions about SCA and public discussions within the community ([[RFC] Making things easier when working with custom products & Simple Content Access (SCA)](https://community.theforeman.org/t/rfc-making-things-easier-when-working-with-custom-products-simple-content-access-sca/31334) in Foreman Community Forum). Are we sure to recommend upstream users to always use SCA? Is there a usecase where it might make sense for them to also disable SCA?

I think the "Satellite only" makes sense if Foreman 3.5+ users *cannot* disable SCA (anymore).

cc @jeremylenz @ianballou @m-bucher

Reviewer 2: I agree. For the time being, entitlement mode remains supported both upstream and downstream. So this section should be included in both doc sets.

Author: If I understand this correctly, I should leave the contents  of the note but make it visible not only for satellite?

Reviewer 2: Right, I'd just remove the `ifdef::satellite[]`.

Author: Got it, thanks!

Author: > Are we sure to recommend upstream users to always use SCA? Is there a usecase where it might make sense for them to also disable SCA?

I am not aware of this usecase; however; we provide the user with the option to disable SCA if they know what they're doing. I think it is okay to generally recommend the new method over the old one, but then again I am not a dev.

. Optional: If you do not wish to enable Simple Content Access (SCA), uncheck the *Simple Content Access* checkbox.
For more information on SCA, see https://access.redhat.com/articles/simple-content-access[Simple Content Access].
+
[NOTE]
====

> We do not recommend disabling SCA as the entitlement mode is deprecated.

Reviewer: ```suggestion
{Team} does not recommend disabling SCA as the entitlement mode is deprecated.
```

Reviewer 2: I think "entitlement mode" is more correct than "the entitlement mode."  I haven't heard of anyone saying it with "the."

--label "_My_Organization_Label_" \
--description "_My_Organization_Description_"
----
+
[NOTE]
====
Organizations created this way have Simple Content Access (SCA) enabled by default.
If you wish to disable SCA, add the `--simple-content-access false` parameter to the command.

> {Team} does not advise you to disable SCA since entitlement mode (not using SCA) is deprecated.

Reviewer: ```suggestion
{Team} does not advise you to disable SCA because entitlement mode (not using SCA) is deprecated.
```

--label "_My_Organization_Label_" \
--description "_My_Organization_Description_"
----
ifdef::satellite[]
+
[NOTE]
====
Organizations created this way have Simple Content Access (SCA) enabled by default.

> If you wish to disable SCA, then add the `--simple-content-access false` parameter to the command.

Reviewer: ```suggestion
If you wish to disable SCA, add the `--simple-content-access false` parameter to the command.
```

> We do not advise you to disable SCA this since the entitlement mode (not using SCA) is deprecated.

Reviewer: ```suggestion
{Team} does not advise you to disable SCA this since the entitlement mode (not using SCA) is deprecated.
```
